org.mixedrealitytoolkit.input/Interactors/Ray/MRTKRayInteractor.cs

```diff
@@ -157,14 +262,33 @@ public override bool isHoverActive
                     bool hoverActive = base.isHoverActive;
                     if (hoverActive)
                     {
-                        if (xrController is ArticulatedHandController handController)
+                        bool isPalmFacingAway = false;
+
+                        #pragma warning disable CS0618 // Type or member is obsolete
+                        #pragma warning disable CS0612 // Type or member is obsolete
+                        if (xrController != null)
+                        {
+                            if (xrController is ArticulatedHandController handController)
+                            {
+                                if (XRSubsystemHelpers.HandsAggregator?.TryGetPalmFacingAway(handController.HandNode, out isPalmFacingAway) ?? true)
+                                {
+                                    hoverActive &= isPalmFacingAway;
+                                }
+                            }
+                        }
+                        else if (HandModel != null)
```

Comment:
Remove warning and null change `MandModel` is not needed

Reply:
Done in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/750/commits/b17f21b3831367c77e4eab113a8e1297ffcca87b, resolving.